extensions/amp-consent/0.1/test/test-amp-consent.js

```diff
@@ -373,7 +376,7 @@ describes.realWin(
             'https://server-test-1/':
               '{"consentRequired": false, "consentStateValue": "unknown", "consentString": "hello"}',
             'https://server-test-2/':
-              '{"consentRequired": true, "consentStateValue": "rejected", "consentString": "mystring", "consentMetadata":{}}',
+              '{"consentRequired": true, "consentStateValue": "rejected", "consentString": "mystring", "consentMetadata":{"consentStringType": 3}}',
```

Comment:
nit: let's use `CONSENT_STRING_TYPE`

Reply:
This is from the server response, not sure if I can use CONSENT_STRING_TYPE here without it being interpreted as a string.

Reply:
I see. I don't know any good way to do that : ( I'm good with using the number.